opentutor_classifier/opentutor_classifier/classifier_dao.py

```diff
@@ -34,8 +34,10 @@ def find_classifier(
         cfac = ClassifierFactory()
         lesson_updated_at = (
             fetch_lesson_updated_at(lesson)
+            # TODO: won't this always resolve to false since the factory is being
+            # created within the function? And therefore a new factory every function call?
```

Comment:
incorrect, the classifier factory checks the file system to see if the model exists. the model itself will be detected regardless of the state of the classifier factory.  Not sure how it would work on serverless execution.